doc/whats_new.rst

```diff
@@ -1,3 +1,19 @@
+X.X.X
+=====
+
+Changelog
+---------
+
+Enhancements
+............
+
+    - A new function :func:`nilearn.regions.extract_regions_labels_img` can be
+      used for extraction of biggest connected components represented as
```

Comment:
Suggestion of simpler sentence: "A new function :func:`nilearn.regions.extract_regions_labels_img` to extract the connected components represented..."